### PR TITLE
HomeWork

### PR DESCRIPTION
**Pull request by Dragoris**
finalized HW_3

### CODE REVIEW

File: matt-Dragoris/hw_3/dice-lab/js/app.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+*/
+ 
+(function rollDice() {
+	var dieOne = document.getElementById('first-die');
+	var dieTwo = document.getElementById('second-die');
+	var rollButton = document.getElementById('roll-dice');
+	
+	rollButton.onclick = function() {
+		var firstRoll = Math.round(Math.random() * (6-1) + 1);
+		var secondRoll = Math.round(Math.random() * (6-1) + 1);
```
Comment (Borgaard) [inline matt-Dragoris/hw_3/dice-lab/js/app.js:30]: Why `(6-1) + 1` and not just 6?

File: matt-Dragoris/hw_4/MonkeyObj.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    this.name = name,
+    this.species = species,
+    this.foodsEaten = [],
+    this.hangry = false
+}
+
+monkey.prototype.eatSomething = function(food){
+    if(this.hangry) {
+        this.hangry = false;
+        console.log (`Thanks for the ${food}, I needed that. *${this.name} is now your friend!*`)
```
Comment (Borgaard) [inline matt-Dragoris/hw_4/MonkeyObj.js:11]: @Dragoris++ for hangry and ES6 string interpolation 😸

File: matt-Dragoris/hw_4/MonkeyObj.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    }
+    this.foodsEaten.push(food);
+}
+
+monkey.prototype.introduce = function() {
+    if(this.foodsEaten.length) {
+        console.log(`Hi! My name is ${this.name} the friendly ${this.species}, and ive eaten some ${this.foodsEaten.join(', ')}`);
+    }else {
+        this.hangry = true;
+        console.log(`Me ${this.name} the enraged ${this.species}, i demand food! FEED ME!!`);
```
Comment (Borgaard) [inline matt-Dragoris/hw_4/MonkeyObj.js:21]: Feeding loco the ape was an absolute rollercoaster, haha!

```
Me loco the enraged ape, i demand food! FEED ME!!
Thanks for the grass, I needed that. *loco is now your friend!*
Hi! My name is loco the friendly ape, and ive eaten some grass, berries
```

**Comment by Dragoris** [comment]
i was just following MDN's blueprint, but yeah, its totally redundant lol 

```
On Saturday, November 19, 2016 3:49 PM, Nicole Borgaard <notifications@github.com> wrote:
```

 @Borgaard commented on this pull request.In matt-Dragoris/hw_3/dice-lab/js/app.js:> +\* get the first die by ID and update the CSS class name to secondDie 
-  (Hint: Use document.getElementById to reference the element, and use the className property to set the name of the CSS class.)
  +
  +3) Create code that runs the function you created whenever the button with the ID value 'roll-dice' is clicked.
-  (Hint: Use document.getElementById to reference the button, and set the value of the onclick property to specify what should happen.)
  +
  +*/
- 
  +(function rollDice() {
-   var dieOne = document.getElementById('first-die');
-   var dieTwo = document.getElementById('second-die');
-   var rollButton = document.getElementById('roll-dice');
-   
-   rollButton.onclick = function() {
-       var firstRoll = Math.round(Math.random() \* (6-1) + 1);
-       var secondRoll = Math.round(Math.random() \* (6-1) + 1);
  Why (6-1) + 1 and not just 6?—

**Comment by Dragoris** [comment]
glad you enjoyed, i thought i could step it up with a constructor function and another small feature 

```
On Saturday, November 19, 2016 3:56 PM, Nicole Borgaard <notifications@github.com> wrote:
```

 @Borgaard commented on this pull request.In matt-Dragoris/hw_4/MonkeyObj.js:> +}
+
+monkey.prototype.eatSomething = function(food){
-    if(this.hangry) {
-        this.hangry = false;
-        console.log (`Thanks for the ${food}, I needed that. *${this.name} is now your friend!*`)
-    }
-    this.foodsEaten.push(food);
  +}
  +
  +monkey.prototype.introduce = function() {
-    if(this.foodsEaten.length) {
-        console.log(`Hi! My name is ${this.name} the friendly ${this.species}, and ive eaten some ${this.foodsEaten.join(', ')}`);
-    }else {
-        this.hangry = true;
-        console.log(`Me ${this.name} the enraged ${this.species}, i demand food! FEED ME!!`);
  Feeding loco the ape was an absolute rollercoaster, haha!Me loco the enraged ape, i demand food! FEED ME!!
  Thanks for the grass, I needed that. _loco is now your friend!_
  Hi! My name is loco the friendly ape, and ive eaten some grass, berries

**Pull request merged**